### PR TITLE
Entreprise network security

### PR DESCRIPTION
**Pull request by krampstudio**

### CODE REVIEW

File: docs/hub/enterprise-hub-network-security.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+To activate this option,
+
+1. Toggle on the "Higher Hub rate-limits" option
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/network-sec-rate-limit.png" alt="Screenshot of the toggle to enable High rate-limits."/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/dark-network-sec-rate-limit.png" alt="Screenshot of the toggle to enable High rate-limits."/>
+</div>
+
+Please note, you're Enterprise Hub Plus subscription will be verified when activating this option.
```
Comment (julien-c) [inline docs/hub/enterprise-hub-network-security.md:35]:
```suggestion
Please note, your Enterprise Hub Plus subscription will be verified when activating this option.
```

File: docs/hub/enterprise-hub-network-security.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+2. Ensure the Organization IP Ranges is defined 
+
+Once defined, higher rate limits will apply to the member of your organization with IPs matching the defined ranges. 
+
+
+## Enforce authenticated access to the Hugging Face Hub
+
+This option will ensure only authenticated users from your organization will be able to access the Hugging Face Hub. All public pages will show the following message if access unauthenticated:
```
Comment (julien-c) [inline docs/hub/enterprise-hub-network-security.md:44]:
```suggestion
This option will ensure that, when browsing from your corporate network, only authenticated users belonging to your organization are able to access the Hugging Face Hub. All public pages will show the following message if access unauthenticated:
```

File: docs/hub/enterprise-hub-network-security.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+</div>
+
+1. Toggle on the "Enforce authenticated access to the Hub" option
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/network-sec-enforce-auth.png" alt="Screenshot of the toggle to enable Enforced authenticated access to the Hub."/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/dark-network-sec-enforce-auth.png" alt="Screenshot of the toggle to enable Enforced authenticated access to the Hub."/>
+</div>
+
+Please note, you're Enterprise Hub Plus subscription will be verified when activating this option.
```
Comment (julien-c) [inline docs/hub/enterprise-hub-network-security.md:58]:
```suggestion
Please note, your Enterprise Hub Plus subscription will be verified when activating this option.
```

**Review by julien-c — APPROVED** [review]
LGTM! 

(nit) also add it to this list (in the index page):

image

And let's share with the sales team when this page is live

File: docs/hub/enterprise-hub-network-security.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/network-sec-ip-ranges.png" alt="Screenshot of the Organization IP Ranges field."/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/dark-network-sec-ip-ranges.png" alt="Screenshot of the Organization IP Ranges field."/>
+</div>
+
+
+## Higher Rate Limits
+
+Apply for higher rate-limits for your organization. 
+
+Most of the actions on the Hub have limits, for example, users are limited to creating to a certain number of repositories per day. This option allows your organization to apply for higher limits for your organization members.
```
Comment (pcuenca) [inline docs/hub/enterprise-hub-network-security.md:24]:
```suggestion
Most of the actions on the Hub have limits, for example, users are limited to creating a certain number of repositories per day. This option allows your organization to apply for higher limits for your organization members. 
```

File: docs/hub/enterprise-hub-network-security.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+To activate this option,
+
+1. Toggle on the "Higher Hub rate-limits" option
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/network-sec-rate-limit.png" alt="Screenshot of the toggle to enable High rate-limits."/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/dark-network-sec-rate-limit.png" alt="Screenshot of the toggle to enable High rate-limits."/>
+</div>
+
+Please note, your Enterprise Hub Plus subscription will be verified when activating this option.
```
Comment (pcuenca) [inline docs/hub/enterprise-hub-network-security.md:35]:
```suggestion
Please, note that your Enterprise Hub Plus subscription will be verified when activating this option.
```

File: docs/hub/enterprise-hub-network-security.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+2. Ensure the Organization IP Ranges is defined 
+
+Once defined, higher rate limits will apply to the member of your organization with IPs matching the defined ranges.
```
Comment (pcuenca) [inline docs/hub/enterprise-hub-network-security.md:39]:
```suggestion
Once defined, higher rate limits will apply to members of your organization whose IPs match the defined ranges. 
```

File: docs/hub/enterprise-hub-network-security.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+1. Toggle on the "Higher Hub rate-limits" option
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/network-sec-rate-limit.png" alt="Screenshot of the toggle to enable High rate-limits."/>
+<img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/enterprise/dark-network-sec-rate-limit.png" alt="Screenshot of the toggle to enable High rate-limits."/>
+</div>
+
+Please note, your Enterprise Hub Plus subscription will be verified when activating this option.
+
+2. Ensure the Organization IP Ranges is defined
```
Comment (pcuenca) [inline docs/hub/enterprise-hub-network-security.md:37]:
```suggestion
2. Ensure the Organization IP Ranges are defined 
```

File: docs/hub/enterprise-hub-network-security.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+Please note, your Enterprise Hub Plus subscription will be verified when activating this option.
+
+2. Ensure the Organization IP Ranges is defined 
+
+
+### Content Access Policy 
+
+You can also define a fine grained Content Access Policy by blocking some section of the Hugging Face Hub. 
+
+For example, you can block your organization's members to access Spaces, by adding `/spaces/*` to the blocked URLs. When users of your organization will navigate to a page that matches the URL pattern they'll be presented the following page:
```
Comment (pcuenca) [inline docs/hub/enterprise-hub-network-security.md:67]:
```suggestion
For example, you can block your organization's members to access Spaces, by adding `/spaces/*` to the blocked URLs. When users of your organization navigate to a page that matches the URL pattern, they'll be presented the following page:
```

**Review by pcuenca — COMMENTED** [review]
some nits

**Pull request merged**